components.js

```diff
@@ -261,6 +261,10 @@ var components = {
 			"title": "HTTP",
 			"owner": "danielgtaylor"
 		},
+		"ichigojam": {
+			"title": "IchigoJam",
+			"owner": "Cocoa Oikawa"
```

Comment:
Please set `owner` to your exact Github username, so that we can link to your page from the Prism download page.